packages/archlinux/.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
lets not add /DS_Store files. We should add this one to gitignore to prevent this from future commits.

Reply:
whoops - added those two patterns to .gitignore